.gitattributes

```diff
@@ -1,6 +1,10 @@
 /tests export-ignore
-/docs export-ignore
+/.tx export-ignore
 /.gitattributes export-ignore
 /.gitignore export-ignore
 /.travis.yml export-ignore
 /.scrutinizer.yml export-ignore
+/*.dist export-ignore
+
+# Line endings
+sake text eol=lf
```

Comment:
This line shouldn't be needed?

Reply:
hmm you're right, I hadn't investigated this line thoroughly while tried to keep the `.gitattributes` file relatively consistent across all the modules

I'll remove it :)